### ISSUE TITLE
Edits on about page through dashboard

### ISSUE THREAD

**Issue by dssong02** [amaidah/ds-portfolio#19]
Is there a way to edit my About page myself through the dashboard?

**Comment by amaidah** [comment]
Yes, but it requires changes to database.  Just paste what you want written here for now.
